docs/getting-started.md

```diff
@@ -1,21 +1,186 @@
+
+Now that we know how to read through messages, let's try filtering to see only messages related to the one we currently have selected. To see this in action you'll want to head back to **All Messages** (<kbd>a</kbd>) before each bullet point. 
+
++ Type <kbd>s</kbd> to narrow to the stream of the current message.
+
++ Type <kbd>S</kbd> to narrow to the topic of the current message. *Note that this works even if you weren't already in the topic's stream.*
+
+You can also type <kbd>z</kbd> to 'zoom in' or 'zoom out.' What do I mean by 'zoom in/out?' Essentially if you're currently in a topic narrow, <kbd>z</kbd> will place you in the selected message's stream, but if you're in a stream <kbd>z</kbd> will place you in the selected message's topic. Try it and see for yourself, press <kbd>z</kbd> a few times and check out your [current narrow](#Your-Current-Narrow).
+
+## Sending Messages
+
+### Reply to a Message
+
+To reply to a message in a narrow (stream/topic/PM), rest your cursor on the message you want to reply to and then type <kbd>r</kbd>. Type your message in the Message box that pops up at the bottom of the middle column. Type <kbd>ctrl</kbd><kbd>d</kbd> to send. If you change your mind and don't want to send the message, type <kbd>esc</kbd> to get out of the message editor. Let's try replying to a random message in the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream (don't worry about messing anything up, the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream was made for stuff like this).
+
+<img src="getting-started-imgs/reply.png" width="75%">
+
+#### Thumbs-up Reaction
+
+If you want to show you agree with the current message, type <kbd>+</kbd> to add a thumbs-up reaction. Type <kbd>+</kbd> again to remove the thumbs-up.
+
+<img src="getting-started-imgs/thumbs-up.png" width="75%">
+
+### Reply via a Private Message
+
+Let's try sending a PM to the author of a message. Select the message you sent to the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream earlier and press <kbd>R</kbd> to send a PM to yourself. Type your message in the message editor that appears at the bottom of the middle column and then type <kbd>ctrl</kbd><kbd>d</kbd> to send. Press the <kbd>P</kbd> hotkey as we did earlier in the tutorial to narrow to your PM's and make sure everything worked properly.
+
+### Send a Private Message to Someone New
+
+You can send a PM by moving your cursor to the list of "Users" in the left column and selecting the name of the person you'd like to send a message to.
+
+If you grabbed the latest unreleased version from [Github](https://github.com/zulip/zulip-terminal) or are running a version greater than 0.6.0 (use `zulip-term --version` to check), you can try out our new autocomplete feature! 
+
+To send a PM using the autocomplete feature:
+1. Type <kbd>x</kbd> from anywhere in Zulip Terminal. A message editor will pop open at the bottom of the middle column. 
```

Comment:
> Type <kbd>x</kbd> from anywhere in Zulip Terminal.

_Almost_ anywhere, right—it won't work if you're focused on the compose box, or the search box, etc. Could just say "use the <kbd>x</kbd> hotkey"; that'd be correct.